data/korath/korath ships.txt

```diff
@@ -8,11 +8,109 @@
+		"drag" 18.3
+		"heat dissipation" .65
+		"fuel capacity" 900
+		"cargo space" 605
+		"outfit space" 827
+		"weapon capacity" 297
+		"engine capacity" 181
+		
+		weapon
+			"blast radius" 300
+			"shield damage" 4600
+			"hull damage" 2300
+			"hit force" 5900
+	outfits
+		"Korath Banisher" 2
+		"Korath Grab-Strike" 3
+		"Korath Warder" 2
+		
+		"Systems Core (Large)"
+		"Triple Plasma Core"
+		"Large Heat Shunt" 3
+		"Fuel Processor"
+		"Korath Repeater Rifle" 190
+		
+		"Steering (Planetary Class)"
+		"Steering (Asteroid Class)"
+		"Thruster (Planetary Class)"
+		"Thruster (Asteroid Class)"
+		"Hyperdrive"
+		"Jump Drive"
+		
+	turret -37 -155 "Korath Banisher"
+	turret 0 -155 "Korath Warder"
+	turret 37 -155 "Korath Grab-Strike"
+	turret -96 0 "Korath Grab-Strike"
+	turret 96 0 "Korath Grab-Strike"
+	turret -80 130 "Korath Warder"
+	turret 80 130 "Korath Banisher"
+	engine -33 171 .4
+	engine -2 182 .6
+	engine 31 171 .4
+	fighter 51 59
+	fighter -51 59
+	fighter 0 -42
+	fighter 0 16
+	explode "tiny explosion" 120
+	explode "small explosion" 60
+	explode "medium explosion" 70
+	explode "large explosion" 50
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "After the Korath were exiled from their homelands, efficient resource collection became a priority. World-ships were too precious to risk on dangerous forays, and Raiders lacked the cargo capacity. The Dredger was born as an answer to this problem, featuring an abundance of cargo and passenger space, with formidable defensive armament. With such a complement, these bulky ships can also offer potent support for frontline Raiders."
+
+
+
+ship "Korath Dredger" "Korath Dredger (Crippled)"
+	sprite "ship/dredger"
+	outfits
+		"Korath Banisher" 2
+		"Korath Grab-Strike" 1
+		"Korath Warder" 1
+		
+		"Triple Plasma Core"
+		"Large Heat Shunt" 3
+		"Fuel Processor"
+		"Korath Repeater Rifle" 190
+		
+		"Steering (Planetary Class)"
+		"Steering (Lunar Class)"
+		"Thruster (Planetary Class)"
+		"Thruster (Comet Class)"
+		"Hyperdrive"
+		"Jump Drive (Broken)"
+		
+	turret -37 -155 "Korath Banisher"
+	turret 0 -155 "Korath Warder"
+	turret 37 -155
+	turret -96 0
+	turret 96 0 "Korath Grab-Strike"
+	turret -80 130
+	turret 80 130 "Korath Banisher"
+	description "After the Korath were exiled, efficient resource collection became a priority, particularly through raiding. Their world-ships were too precious to risk on dangerous forays, and Raiders lacked the capacity. The Dredger was their first new ship: abundant cargo and passenger space, bays for four fighters, and seven turrets. With such a complement, these bulky ships are formidable supports for frontline Raiders."
+	description "This Dredger has taken significant damage, and is missing several key systems, such as its system core and some of its weaponry."
+
+
+
 ship "Korath Raider"
 	sprite "ship/raider"
 	thumbnail "thumbnail/raider"
 	attributes
-		category "Heavy Warship"
+		category "Medium Warship"
```

Comment:
Why demote the Raider?

Reply:
It's felt a long time coming, especially with the Shield Beetle, a tier 1.25 ship, being able to got toe-to-toe with it. It's a high mass ship, but so are the other Korath automata medium warships.

Reply:
It isn't a demotion. Rather, it's a  promotion from "mediocre heavy" up to "great medium."

More seriously, though, it's more of a preparatory step to get people used to the idea that there's going to be a range of new Korath ships coming. Traditionally speaking, most factions have just one or sometimes two ships for each category. The Raider is a relatively fast raiding ship, which is a role more befitting the title "medium warship" than heavy. 

This is also meant to be sort of a subtle threat for the player, too. The fleets raiding the Remnants? Medium warships and occasionally Heavy Freighters that provides pretty good combat support. Most players will naturally assume that there is a heavy warship out there that they haven't seen yet. And eventually, they'll be right. Even if it isn't the case right now, the fact that it is "only" a medium will keep players on their toes about what else might be out there.

In contrast, if they see a heavy warship with a heavy freighter, well, "heavy warship" is the top end of the combat scale. So players generally feel safe because, well, they've been killing Raiders since they arrived here, and that's the Korath heavy warship, so all's good, right?

Reply:
> More seriously, though, it's more of a preparatory step to get people used to the idea that there's going to be a range of new Korath ships coming.

That's fair, but I don't see how it's related to this PR. IMO it would make more sense to move that change into another forthcoming one that actually adds that additional variety. This will also end the potential argument from keeping it here.